azure_functions_worker/dispatcher.py

```diff
@@ -381,6 +382,7 @@ def _serialize_exception(self, exc):
 
             # Reload azure namespace for customer's libraries
             importlib.reload(sys.modules['azure'])
+            importlib.reload(sys.modules['azure.functions'])
```

Comment:
did you test this e2e? i.e - write a python function that imports a different version of azure.funcitons package

Also, add logging 
-  'Reloading azure` also, include version of the package before and after reload
- 'Reloading azure-functions`- same here include version of the package before and after reload
here that has info the version that is loaded.

Reply:
Made changes to add logging for when we reload both modules. `azure` itself isn't guaranteed to a have a version or a well defined path, but I've added specific path information for when we reload `azure.functions` so that we can inspect if the path has change from the system to the user path. There is also handling if we are unable to reload `azure.functions` (for when the user hasn't included it). This ensures that we are able to fall back to the default version. 

I was able to validate these changes end to end for the following app scenarios: 

1. No dependencies
2. `azure` dependency 
3. `azure.functions` included in user code and additional `azure` dependencies to see if we are able to resolve properly
4. `azure` native dependency